downstream/modules/platform/proc-determine-hub-route.adoc

```diff
@@ -0,0 +1,13 @@
+[id="proc-determine-hub-route_{context}"]
+
+= Determining the {HubName} Route
+
+Use the following procedure to determine the hub route.
+
+. Procedure
+
+. Navigate to menu:Networking[Routes].
+. Select the project you used for the install.
+. Copy the location of the `private-ah-web-svc` service.
+Note that the name of the service is different if you used a different name when creating the {HubName} instance.
```

Comment:
If this is a note then make it a note. Otherwise I'd suggest just "The name of the service matches the name you specify when you create the {HubName} instance."